### ISSUE TITLE
Using ITRANS as input_encoding errors out

### ISSUE THREAD

**Issue by poojapi** [kmadathil/sanskrit_parser#136]
I'm using the parser in the tests as follows -> 

`lexan = Parser(input_encoding='ITRANS', replace_ending_visarga='s')` 

But getting the following error while parsing 

```
File "generate_bg_Lexical_Analyzer_pass_fail.py", line 156, in test_splits
    graph = lexan.parse(f)
  File "/Users/pooja/personal/experiments/parser/test/lib/python3.8/site-packages/sanskrit_parser-0.1.0-py3.8.egg/sanskrit_parser/api.py", line 98, in parse
    self.input_encoding = SCHEMES[self.input_encoding]
KeyError: 'itrans'
```

When I tried changing the code in api.py to use itrans in caps like this -> `self.input_encoding = SCHEMES[self.input_encoding.upper()]`, I did not get any error.

**Comment by kmadathil** [comment]
use `input_encoding='itrans'` instead.

@avinashvarna , could you comment on the choice of lowercase convention for the api?

**Comment by avinashvarna** [comment]
We can remove the `dataclass` annotation and add an `__init__` function to the Parser. I will make this change to keep it simple.

**Comment by kmadathil** [comment]
@avinashvarna - if the fix above is not merged yet, can you cherry pick that into master?

**Comment by avinashvarna** [comment]
Applied the patch to master.

**Comment by avinashvarna** [comment]
Can you please push an update to pypi if everything works as expected? Maybe also fix #141 while at it?

**Comment by kmadathil** [comment]
Done.
